src/login/data/reducers.js

```diff
@@ -47,6 +47,7 @@ const reducer = (state = defaultState, action = {}) => {
         loginResult: action.payload,
       };
     case LOGIN_REQUEST.FAILURE:
+      // eslint-disable-next-line no-case-declarations
```

Comment:
Having variables in case like this cause the variable being out of scope of this particular case. Therefore it is discouraged.

instead of disabling the eslint here, can we do like this?

```
    case LOGIN_REQUEST.FAILURE: {
      const { email, loginError, redirectUrl } = action.payload;
      return {
        ...state,
        loginErrorCode: loginError.errorCode,
        loginErrorContext: { ...loginError.context, email, redirectUrl },
        submitState: DEFAULT_STATE,
      };
    }

Reply:
How is it different from the code I have already written?